### PR TITLE
release: provide `install` script to install binary

### PR DESCRIPTION
**Pull request by Pothulapati**
This PR adds a new `install` script that will be part
of `gh-pages`, through which users will be able to install
the `linkerd-smi` binary.

After this PR is approved and before this is merged,
`install` will be moved into `gh-pages` and users should
be able to install the binary by running

```bash
curl -sL https://linkerd.github.io/linkerd-smi/install | sh
```

The latest release version will be updated in the script, by
the release workflow

Signed-off-by: Tarun Pothulapati <tarunpothulapati@outlook.com>

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -172,8 +176,10 @@ jobs:
         helm repo index --merge index.yaml build-archives/helm
         mv build-archives/helm/index.yaml ./index.yaml
         cp -r build-archives/helm/. ./
+        # update version in install script
+        sed -i 's/LINKERD_SMI_VERSION:-.*/LINKERD_SMI_VERSION:-"${{needs.gh_release.outputs.tag}}"}/' ./install
         git config user.name github-actions
         git config user.email github-actions@github.com
-        git add linkerd-smi-*.tgz index.yaml
+        git add linkerd-smi-*.tgz index.yaml install
         git commit -sm "Add new Helm Chart"
```
Comment (alpeb) [inline .github/workflows/release.yml]: Now that we have access to the tag, can you make this commit message more explicit by including that tag, like `"Add new Helm Chart vx.xx"`?

**Review by alpeb — APPROVED** [review]
LGTM :+1: 
Just one small nit.

File: .github/workflows/release.yml (hunk 1 above)
Comment (Pothulapati) [reply .github/workflows/release.yml]: yep, good idea!

**Comment by Pothulapati** [comment]
@alpeb  Also, Moved the `install` script form this PR into https://github.com/linkerd/linkerd-smi/tree/gh-pages branch tagged at `v0.1.0`.

You should now be able to try the install, by running

```
url -sL https://linkerd.github.io/linkerd-smi/install | sh   
```

**Comment by alpeb** [comment]
Yep, that did the trick! :taco: :taco:

**Pull request merged**